components/CheckDocumentation/connections/BlockingQuery.tsx

```diff
@@ -0,0 +1,129 @@
+        <code>{config.settings["critical_blocked_age_secs"]}</code> seconds.
+        Resolves automatically once these queries stop meeting the criteria.
+      </p>
+      <p>
+        Ignores any blocking queries that contain the{" "}
+        <code>/* pganalyze:no-alert */</code> or{" "}
+        <code>/* pganalyze=no-alert */</code> magic comment.
+      </p>
+    </>
+  );
+};
+
+const BlockingQueryGuidance: React.FunctionComponent<CheckGuidanceProps> = ({
+  urls: { backendsUrl, SettingLink },
+  issue,
+}) => {
+  const Link = useSmartAnchor();
+  const issueDetails = issue && JSON.parse(issue.detailsJson);
+  const showAutovacuum = issueDetails?.["has_autovacuum_to_prevent_wraparound"];
+  return (
+    <div>
+      <h4>Impact</h4>
+      <p>
+        Queries that are blocking many other queries for certain amount of time
+        likely holding some strong locks. These queries can block some trivial
+        SELECT queries sometimes, which can cause the whole system down. They
+        can also prevent VACUUM (including autovacuum) from cleaning up dead
+        rows, leading to table or index bloat. They may also be consuming a lot
+        of I/O or CPU resources, impacting overall performance.
+      </p>
+      <h4>Common Causes</h4>
+      <ul>
+        {showAutovacuum && (
+          <li>
+            <h5>Autovacuum (to prevent wraparound)</h5>
+            <p>
+              In general, autovacuum workers don't block other queries. Even if
+              the other query tries to acquire a lock that conflicts with the
+              running autovacuum, lock acquisition will interrupt the
+              autovacuum. However, if the autovacuum is running to prevent
+              transaction ID wraparound, the vacuum won't be interrupted and
+              will block other queries trying to acquire conflicting locks.
+            </p>
+          </li>
+        )}
+        <li>
+          <h5>Locks</h5>
+          <p>
+            The queries may be holding some strong locks. Check the{" "}
+            <Link to={backendsUrl}>Connection Traces</Link> page for the queries
+            these queries are blocking. You can check{" "}
+            <strong>Wait Events</strong> page for the queries waiting for these
+            blocking queries, and look for any wait events of type{" "}
+            <strong>Lock</strong>. If you have{" "}
+            <SettingLink setting="log_lock_waits" /> turned on, you can also see
+            details about locking in the <strong>Logs</strong> page for these
+            queries.
+          </p>
+          <p>
+            <code>ACCESS EXCLUSIVE</code> lock xis the strongest lock that can
+            even block a <code>SELECT</code> query. This lock is acquired by the
+            commands like <code>VACUUM FULL</code>. One of the table-rewriting
+            variants of <code>ALTER TABLE</code> also acquire a lock at this
+            level.
+          </p>
```

Comment:
I don't follow how this paragraph connects to the rest of this cause.

Reply:
yeah I wanted to add something here but I do find it hard to understand too now that I read it. I removed this